components/nds-panel/__tests__/__snapshots__/Panel.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Panel should match snapshot for default panel 1`] = `
 <div
-  className="panel"
+  className="panel panel--supporting"
```

Comment:
This doesn't feel right to me. I know our default panel is a supporting one but we don't actually have this modifier in the CSS so not sure we should have this class.